share/doc/xml/srcflux.xml

```diff
@@ -69,11 +68,30 @@
 	psfmethod argument. Extended sources should use psfmethod=ideal,
 	which makes no correction, or psfmethod=psffile and provide
     an image of the surfrace brightness as the psffile parameter.
-        The other options are discussed below in the psfmethod parameter 
+        The other options are discussed below in the psfmethod parameter
      and the "PSF Correction" section.
       </PARA>
+      <PARA title='A fully Baysian computation'>
+	This tool is based on aprates, which performs a fully Baysian computation,
+	thus the result is not a (frequentist) confidence interval,
+	but should correctly be called credible interval.
+	However, given the common usage in astronomy, this tool uses the term CONF
+	as parameter and in the output.
+      </PARA>
+      <PARA title='Upper limits vs. upper value for flux uncertainty'>
+	If the lower bound for the uncertainty of the flux reaches 0, only the upper
+	bound is reported. This number is sometimes erroneously used as the upper flux
+	limit for an undetected source, but strictly speaking the uncertainty on a flux
+	measurement and the upper limit of an undetected source are two different questions,
+	even though the numbers often come out to be in a similar range. When talking about the
+	upper limit for an undetected sources (the maximal flux that a source could have without
+	being detected), a single number for conf is not sufficient. Instead, two numbers are needed:
+	The probability of detecting a source in background fluctuations (false positive) and the
+	probability of missing a real source because it is compatible with background (false negative).
+	This calculation is beyond scope of this tool.
+      </PARA>
```

Comment:
Perhaps we should just quote the exact paragraph from Kashyap et al 2010 which has a very good description of the problem in Sec.2.1, or slightly shorter version, or just refer to this paragraph in the text. 

I would rephrase the current text and just state:
"The tool does not return upper limit and that the reported values are always calculated for lower and upper confidence bound. The upper limit calculation requires a detection threshold which does not dependent on the number of counts or the source flux. The detection threshold is not used in this tool. See Kashyap et al 2010, for full discussion of the difference between confidence bounds and upper limits."